src/Sign.Core/SignatureProviders/RSAPKCS1SHA256SignatureDescription.cs

```diff
@@ -2,9 +2,11 @@
 
 namespace Sign.Core
 {
-    internal sealed class RSAPKCS1SHA256SignatureDescription : RSAPKCS1SignatureDescription
+    // This type and its default constructor are public because:
+    //   "Algorithms added to CryptoConfig must be accessible from outside their assembly."
```

Comment:
Seems like a requirement of this type? https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.cryptoconfig.addalgorithm?view=net-7.0 (based on exception message). A URL reference could be nice.